tests/swarm/test_reliable_delivery.py

```
@@ -0,0 +1,573 @@
+        config, agent_id = create_config()
+        bus = create_bus(config)
+        
+        delivery = ReliableDelivery(bus, agent_id)
+        
+        # Mock bus with 10% packet loss
+        async def publish_with_loss(*args, **kwargs):
+            pass  # Simulate publish
+        
+        bus.publish = AsyncMock(side_effect=publish_with_loss)
+        
+        successful = 0
```

Ruff (0.14.10)

Local variable `successful` is assigned to but never used

Remove assignment to unused variable `successful`

(F841)